src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckTest.java

```diff
@@ -142,7 +142,7 @@ public void testOrderOfProperties() throws Exception {
 
     @Test
     public void testAcceptableTokensMakeSense() {
-        final int expectedTokenTypesTotalNumber = 169;
+        final int expectedTokenTypesTotalNumber = 170;
```

Comment:
urelated to this line.

I do not see removal of https://github.com/checkstyle/checkstyle/pull/8289/files#diff-e70aa0384ec275f925183489857b630dR151

Reply:
Fixed

Reply:
I do not see changes.

Reply:
Changes pushed now.